COMETwebapp/Shared/Header.razor

```diff
@@ -99,4 +119,13 @@
         if (_offCanvas != null)
             await _offCanvas.ToggleAsync();
```

Comment:
private fields do not start with an underscore "_", please rename and prefix with this: `this.offCanvas`.